lib/ansible/module_utils/gcp.py

```diff
@@ -130,9 +123,9 @@ def _get_gcp_libcloud_credentials(service_account_email=None, credentials_file=N
     if service_account_email is None or credentials_file is None:
         try:
             import secrets
-            display.deprecated(msg=("secrets file found at '%s'.  This method of specifying "
-                                    "credentials is deprecated.  Please use env vars or "
-                                    "Ansible YAML files instead" % (secrets.__file__)), version=2.5)
+            module.deprecate(msg=("secrets file found at '%s'.  This method of specifying "
```

Comment:
@erjohnso, yes, exactly that. [ It was added ](https://github.com/ansible/ansible/blob/f217dae938dd451d32c54643ae98020cc592e63d/lib/ansible/module_utils/basic.py#L781)within the last few months.